galgebra/mv.py

```diff
@@ -1719,16 +1716,16 @@ def Dop_mv_expand(self, modes=None):
         terms = list(zip(coefs, bases))
         return sorted(terms, key=lambda x: self.Ga.blades.flat.index(x[1]))
 
-    def Dop_str(self):
+    def Dop_str(self, print_obj):
         if len(self.terms) == 0:
             return ZERO_STR
 
         mv_terms = self.Dop_mv_expand(modes=simplify)
         s = ''
 
         for sdop, base in mv_terms:
-            str_base = printer.latex(base)
-            str_sdop = printer.latex(sdop)
+            str_base = print_obj.doprint(base)
+            str_sdop = print_obj.doprint(sdop)
```

Comment:
These lines are the ones responsible for flipping the output from latex to string. This method is `Dop_str`, so it should not have been producing `latex`. That's what `Dop_latex_str` is for.